plugin/plugin.go

```diff
@@ -94,6 +102,7 @@ func cmdDel(args *skel.CmdArgs) error {
 	if err := rpcCall("Infoblox.Release", extArgs, &result); err != nil {
```

Comment:
Nit picking....

For consistency and readability, you might want to consider doing this:
`result := &struct{}{}`
....
`if err := rpcCall("Infoblox.Release", extArgs, result); err != nil {`